build/Community.Toolkit.Common.props

```diff
@@ -32,4 +32,19 @@
     <ContinuousIntegrationBuild>$(TF_BUILD)</ContinuousIntegrationBuild>
   </PropertyGroup>
 
-</Project>
+  <PropertyGroup>
+    <!-- Common Outputs -->
+    <BuildDir>~build\</BuildDir>
+    <PublishDir>~publish\</PublishDir>
```

Comment:
Can we keep the build outputs in the project directory or move them to repository directory? If we prefer a common output path, we can move them to repository root.

```suggestion
    <!-- Common Outputs -->
    <BuildDir>$(RepositoryDirectory)~build\$(MSBuildProjectName)</BuildDir>
    <PublishDir>$(RepositoryDirectory)~publish\$(MSBuildProjectName)</PublishDir>
```